transactions/src/main/java/org/jboss/as/txn/service/JTAEnvironmentBeanService.java

```diff
@@ -56,7 +57,9 @@ public void start(StartContext context) throws StartException {
         // recovery nodes
         jtaEnvironmentBean.setXaRecoveryNodes(Collections.singletonList(nodeIdentifier));
         // setup the XA orphan filters
-        jtaEnvironmentBean.setXaResourceOrphanFilterClassNames(Arrays.asList(JTATransactionLogXAResourceOrphanFilter.class.getName(), JTANodeNameXAResourceOrphanFilter.class.getName(), SubordinateJTAXAResourceOrphanFilter.class.getName()));
+        if (System.getProperty("JTAEnvironmentBean.xaResourceOrphanFilterClassNames") == null) {
```

Comment:
I take what you are saying, but how about if the list was wanted to set to empty list e.g. -DJTABean.orphanFilters=""
if (cname.size() == 0 and System.getProperty("Bean.filters") == null)